### PR TITLE
Add gpu acceleration support for editor helper functions

### PR DESCRIPTION
**Pull request by vivekrj1806**
Test: ./ultrahdr_unit_test

Change-Id: Ifb4ada9156863bdbd3c5de54a07a1a4cdb1e2fb8

### CODE REVIEW

File: lib/include/ultrahdr/editorhelper.h [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 #include "ultrahdr_api.h"
 #include "ultrahdr/ultrahdrcommon.h"
+#include "ultrahdr/jpegr.h"
```
Comment (DichenZhang1) [inline lib/include/ultrahdr/editorhelper.h:22]: I think this is not needed. Could you double check?

Comment (vivekrj1806) [reply lib/include/ultrahdr/editorhelper.h:22]: This is needed since we are making use of isBufferDataContiguous function declared in jpegr.h

Comment (ram-mohan) [reply lib/include/ultrahdr/editorhelper.h:22]: @vivekrj1806  as editorhelper.h functionality is mostly isolated, it does not make sense for it to include jpegr.h, If you move is isBufferDataContiguous inside ultrahdrcommon.h then there is no need to include jpegr.h

Comment (vivekrj1806) [reply lib/include/ultrahdr/editorhelper.h]: Done.

**Pull request merged**